litmus-portal/frontend/public/locales/en/translation.yaml

```diff
@@ -91,7 +91,7 @@ workflowStepper:
   back: Back
 
 editor:
-  status: 'YAML Status:'
+  status: "YAML Status:"
```

Comment:
May you revert the quotes changes are not required. They might have been added automatically on save. Please prefer reverting it.

Reply:
Same for all similar changes

Reply:
Sure